src/QMCDrivers/DMC/DMCFactoryNew.cpp

```diff
@@ -35,7 +36,7 @@ QMCDriverInterface* DMCFactoryNew::create(const ProjectData& project_data,
   DMCDriverInput dmcdriver_input;
   dmcdriver_input.readXML(input_node_);
   QMCDriverInterface* qmc = new DMCBatched(project_data, std::move(qmcdriver_input), global_emi,
```

Comment:
Since we're modifying adjacent code, what do you think about updating this to use std::make_unique and return a unique_ptr.

Reply:
I think all of XMCFactory/XMCFactoryNew::create raw pointers needs to be updated. It is a different PR topic. May be you can scan through the code?